android/src/main/java/com/thebluealliance/androidclient/binders/TeamInfoBinder.java

```diff
@@ -118,6 +125,13 @@ public void updateData(@Nullable TeamInfoBinder.Model data) {
             string.setSpan(new TextAppearanceSpan(mActivity,
                     R.style.InfoItemLabelStyle), 0, 3, Spannable.SPAN_INCLUSIVE_INCLUSIVE);
             teamFullName.setText(string);
+            teamFullName.setMaxLines(TEAM_FULL_NAME_COLLAPSED_MAX_LINES);
+            teamFullName.setEllipsize(TextUtils.TruncateAt.END);
```

Comment:
So many questions about this on SO! setEllipsize() sounds fragile, interacting with other TextView properties like setSingleLine(), setMaxLines(), setLines(), setHorizontallyScrolling(), ... and varying by Android release.

Oh: [setEllipsize()](http://developer.android.com/reference/android/widget/TextView.html#setEllipsize(android.text.TextUtils.TruncateAt)) says

> If setMaxLines(int) has been used to set two or more lines, only END and MARQUEE are supported (other ellipsizing types will not do anything).

If the current code works on older Android releases, just drop my idea to move the ellipses.

http://stackoverflow.com/search?q=setEllipsize+is%3Aquestion+%5Bandroid%5D -- 33 questions on `setEllipsize()`
http://stackoverflow.com/search?q=ellipsize+is%3Aquestion+%5Bandroid%5D -- 429 questions on `ellipsize`